### ISSUE TITLE
Train my own dataset

### ISSUE THREAD

**Issue by astaroth106** [thtrieu/darkflow#266]
I've trying to train a yolov2 model with only 1 class, but when I finish training it and test it I get predictions all over the image. 
![hive012 0](https://cloud.githubusercontent.com/assets/7307397/26617669/00976e0c-459c-11e7-9fa9-befa62e20e4d.jpg)

(All the other test images have the same prediction boxes)

This is my command:
flow --model cfg/beeModel.cfg --train --gpu 0.9

This is my cfg file:
[net]
# Testing
batch=1
subdivisions=1
# Training
batch=64
subdivisions=8
width=608
height=608
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1

learning_rate=0.001
burn_in=1000
max_batches = 500200
policy=steps
steps=400000,450000
scales=.1,.1

[convolutional]
batch_normalize=1
filters=32
...
[convolutional]
size=1
stride=1
pad=1
filters=30
activation=linear

[region]
anchors =  0.57273, 0.677385, 1.87446, 2.06253, 3.33843, 5.47434, 7.88282, 3.52778, 9.77052, 9.16828
bias_match=1
classes=1
coords=4
num=5
softmax=1
jitter=.3
rescore=1

object_scale=5
noobject_scale=1
class_scale=1
coord_scale=1

absolute=1
thresh = .1
random=1

I have step 5500 - loss 3.702622175216675 - moving ave loss 3.791347478027358

Do I need to train the model further or what should I do?

**Comment by EmmanouelP** [comment]
Yeah man i dont believe you have converged yet.See if you can bring loss at around 0.

**Comment by astaroth106** [comment]
Yes I just needed to be more patient, the model is now a 10000 steps and it is working well.

**Comment by Xianlei** [comment]
@astaroth106  I have the same problem here:

step 2000, epoch 43,    loss ~2.0

I am training on small dataset with 480 images with one  class. I use pre-trained weights tiny-yolo-voc....

Any suggestion ? thanks

**Comment by bauerzhou** [comment]
@Xianlei 
I think you should keep training  until the loss less than 1 and take a predict test ..

**Comment by jjjjohnson** [comment]
Hi @astaroth106
Could you explain the `anchors = 0.57273, 0.677385 ...` mean for bonding box? For example, what is anchors = 1, 1 corresponding to the size in the real image
